software/coolify.yml

```diff
@@ -0,0 +1,15 @@
+
+name: "Coolify"
+website_url: "https://coolify.io/"
+source_code_url: "https://github.com/coollabsio/coolify"
+description: "An open-source & self-hostable Heroku / Netlify / Vercel alternative."
+licenses:
+  - Apache-2.0
+platforms:
+  - PHP
+  - Docker
+tags:
+  - Automation
+  - Deployment
+  - Web Hosting
```

Comment:
```suggestion
```
These tags don't unfortunately exist, but automation is sufficient enough. Otherwise, looks all good to me as soon as changes are made.

Reply:
Should I remove the previous tags and include Automation . Or keep it as it is with suggested changes

Reply:
Adjusted the Tags